ra_aid/agent_utils.py

```
@@ -56,58 +61,189 @@
+    Returns:
+        Total estimated token count
+    """
+    if not messages:
+        return 0
+
+    estimate_tokens = CiaynAgent._estimate_tokens
+    return sum(estimate_tokens(msg) for msg in messages)
+
+
+def state_modifier(
+    state: AgentState, max_tokens: int = DEFAULT_TOKEN_LIMIT
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)